onedal/kmeans_init/tests/test_kmeans_init.py

```diff
@@ -0,0 +1,93 @@
+# ===============================================================================
+# Copyright 2023 Intel Corporation
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# ===============================================================================
+
+import pytest
+import numpy as np
+from numpy.testing import assert_allclose, assert_array_equal
+from daal4py.sklearn._utils import daal_check_version, sklearn_check_version
+
+if daal_check_version((2023, 'P', 200)):
+    from onedal.kmeans_init import kmeans_plusplus
+    from onedal.tests.utils._device_selection import get_queues
+
+    #from sklearn.cluster import KMeans
+    from onedal.kmeans import KMeans
+    from sklearn.datasets import load_breast_cancer
+    from sklearn.metrics import davies_bouldin_score
+    from sklearn.model_selection import train_test_split

```

Comment:
Imports which don't require version check should be moved to header

Reply:
Partially fixed